conan/api/subapi/remotes.py

```diff
@@ -40,12 +40,12 @@ def reinit(self):
 
     def list(self, pattern=None, only_enabled=True):
         """
-        Obtain a list of ``Remote`` objects matching the pattern.
+        Obtain a list of :ref:`Remote <conan.api.model.Remote>` objects matching the pattern.
```

Comment:
This way the docstring in the documentation links to the remote. Note that this already happens for those methods that have typing and reference the Remote object directly